features/webgl2-context-powerpreference.yml

```diff
@@ -0,0 +1,5 @@
+name: WebGL2 `powerPreference` parameter
```

Comment:
Has similar support as WebGL1 `powerPreference`, with Safari being significantly different (10.1/3 vs 15). We could merge these two, but based on the significant difference, I'd lean towards no.